src/acktest/k8s/resource.py

```diff
@@ -340,6 +340,56 @@ def get_resource_condition(reference: CustomResourceReference, condition_name: s
 
     return None
 
+
+def condition_exists(reference: CustomResourceReference, condition_name: str):
+    """
+    Check if the specified condition in .status.conditions exists. Can return one of three values as specified below.
+
+    Returns:
+        None if the resource cannot be found or does not have .status.conditions field at all
+        False if the resource exists and has a .status.conditions field but does not contain the desired condition
+        True if the resource exists and has a .status.conditions field and contains the desired condition
+    """
+
+    if not get_resource_exists(reference):
+        return None
+
+    resource = get_resource(reference)
+    if 'status' not in resource or 'conditions' not in resource['status']:
+        return None
```

Comment:
Yes that's correct, unless I'm mistaken then `status.conditions` should always exist as an empty list when `status` exists (`rm.setStatusDefaults`). So if `status` doesn't exist or `status.conditions` doesn't exist, that indicates something has gone wrong. 

I guess a longer description could be: given the existence of `status.conditions`, return the existence of a condition. Otherwise, throw an exception (per your above suggestion).